mm2src/coins/utxo/utxo_tests.rs

```diff
@@ -2641,6 +2642,32 @@ fn test_get_sender_trade_fee_dynamic_tx_fee() {
     assert_eq!(fee1, fee3);
 }
 
+#[test]
+// validate an old tx with no output with the burn account
+// TODO: remove when we disable such old style txns
```

Comment:
please move notes above the [test] attribute

Reply:
fixed in [b84b3a9](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2112/commits/b84b3a9213f9585d263fd996424818cd976126e7)